packages/elements/package.json

```diff
@@ -11,7 +11,7 @@
     "test": "npm run test:unit && npm run test:integration",
     "test:unit": "karma start --single-run",
     "test:unit:debug": "karma start --browsers ChromeHeadlessDebug",
-    "test:integration": "mocha --forbid-only --config ./test/integration/.mocharc.jsonc",
+    "test:integration": "mocha --forbid-only --exit --config ./test/integration/.mocharc.jsonc",
```

Comment:
I think this `SseTestServer` should have a `close` method that closes the underlying HTTP server (if `listen()` was called). Use that in `afterEach`. That way you don't have to rely on the `--exit` (which is kind of a dirty hack).

Reply:
Ah, I was wondering why they were still pending.. this explains a lot 😅